05_run_ocp.sh

```diff
@@ -5,6 +5,11 @@ set -e
 source ocp_install_env.sh
 source common.sh
 
+# Switch NetworkManager to internal DNS
+sudo mkdir -p /etc/NetworkManager/conf.d/
+echo -e "[main]\ndns=dnsmasq" | sudo tee /etc/NetworkManager/conf.d/dnsmasq.conf
+systemctl restart NetworkManager
```

Comment:
This should include a test for the file as we don't need to do this every time we run the installer?

Relatedly this should probably go in 02_configure_host.sh where all the other host configuration happens?

Reply:
Also we need sudo for the systemctl restart